metaflow/plugins/aip/aip.py

```diff
@@ -1051,6 +1051,9 @@ def _set_container_labels(self, container_op: ContainerOp):
         container_op.add_pod_label(
             "tags.ledger.zgtools.net/ai-step-name", container_op.name
         )
+        container_op.add_pod_label(
+            "zodiac.zillowgroup.net/product", "batch"
+        )
```

Comment:
it's best to move this to `_get_flow_labels()`
Then it'll be applied to both the container and Workflow level (and the injection of the label wouldn't be needed in `trigger_workflow_template`)

Reply:
@tmckay maybe this is the better spot to add labels. Can you confirm the change and test it so we can get this change merged?